README.md

```diff
@@ -168,7 +141,4 @@ I acknowledge the Author of my design; Cindy Shin in Behance. The master behind
 
 This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [x] The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. However, for this particular one, the link is broken and therefore other developers can't be able to access the terms and conditions. Kindly ensure to fix this 👍